metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -2300,12 +2305,15 @@ class MetalsLanguageServer(
       item <- dependencySources.getItems.asScala
       scalaTarget <- buildTargets.scalaTarget(item.getTarget)
       sourceUri <- Option(item.getSources).toList.flatMap(_.asScala)
+      path = sourceUri.toAbsolutePath
+      _ = buildTargets.addDependencySource(path, item.getTarget)
+      _ = jdkSources.foreach(jdk =>
+        buildTargets.addDependencySource(jdk, item.getTarget)
```

Comment:
nit
```suggestion
      _ = jdkSources.foreach(source =>
        buildTargets.addDependencySource(source, item.getTarget)
```
Calling that `jdk` may be a bit confusing.